packages/amplify-frontend-flutter/lib/configuration-manager.js

```diff
@@ -30,6 +30,21 @@ function normalizeInputParams(context) {
   context.exeInfo.inputParams[constants.Label] = inputParams;
 }
 
+export function displayFrontendDefaults(context) {
+  context.print.info(`| App type: flutter`);
+  context.print.info(`| Configuration file location: ${constants.defaultResDir}`);
+}
+
+export function setFrontendDefaults(context) {
```

Comment:
I think it makes sense to allow the plugin to set context info since it's consistent with the frontend plugins setting the configuration info in the standard flow